src/entity/Answer.java

```diff
@@ -0,0 +1,54 @@
+package entity;
+
+import java.sql.Timestamp;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.Table;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@NamedQueries({
+	@NamedQuery(
+			name = "findAllAnswerInfo",
+			query = "SELECT a FROM Answer AS a ORDER BY a.id DESC"
```

Comment:
```suggestion
			query = "SELECT a.* FROM Answer AS a ORDER BY a.id DESC"
```
aliasのaのみだと何もとれない可能性あるのでこっちに。
テーブル名はcorrect_answersでなくAnswerにしたんでしょうか…？